hpobench/benchmarks/ml_mmfb/entry_point.py

```diff
@@ -0,0 +1,24 @@
+from hpobench.benchmarks.ml_mmfb.histgb_benchmark import HistGBSearchSpace0Benchmark, HistGBSearchSpace1Benchmark, \
```

Comment:
Do we need an extra .py file here? 
Doing something like `this` should make all the available classes accessible under `hpolib.benchmarks.ml` or `hpolib.benchmarks.ml_mmfb`? 
So a `from hpolib.benchmarks.ml import *` should suffice when using the benchmark right?

Reply:
Normally yes. But I need this for the container interface.

Reply:
Can we then also have `__init__` populated as normal?